spec/reek/configuration/configuration_file_finder_spec.rb

```diff
@@ -16,44 +16,44 @@
 
     it 'returns the file in current dir if config_file is nil' do
       options = double(config_file: nil)
-      current = Pathname.new('spec/samples')
+      current = Pathname('spec/samples')
```

Comment:
I suppose it's a matter of taste, but I'm not very fond of methods that look like constants, so I actually prefer `Pathname.new` here.